src/diffpy/srfit/fitbase/recipeorganizer.py

```diff
@@ -730,7 +730,7 @@ def clearConstraints(self, recurse=False):
             self.unconstrain(*self._constraints)
 
         if recurse:
-            f = lambda m: hasattr(m, "clearConstraints")
+            f = lambda m: hasattr(m, "clearConstraints")  # noqa: E731
```

Comment:
Here it may be more readable again if we don't assign the lamda but put it directly into the filter.